### PR TITLE
plugin.audio.ilovemusic - Version 3.0.1

### PR DESCRIPTION
**Pull request by adiko01**
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team
Last Addonupdate for Kodi Leia with an minimalistic update function.

### CODE REVIEW

File: plugin.audio.ilovemusic/resources/senderliste.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+https://streams.ilovemusic.de/iloveradio1.mp3,I LOVE RADIO,https://raw.githubusercontent.com/adiko01/plugin.audio.ilovemusic/Daten/Senderlogos/i_love_radio.png,
```
Comment (basrieter) [inline plugin.audio.ilovemusic/resources/senderliste.txt:1]: Please use a resource add-on for this type of content. You can put the logos in that one and use them from this add-on. The images should either be:

* in this add-on
* in another resource add-on within Kodi
* on their original web locations

Having them somewhere else might introduce issues like images being updated outside the add-on. In worst case scenarios to ones that might be offensive to some people.

**Review by basrieter — CHANGES_REQUESTED** [review]
Please see my comments on the external images.

**Comment by basrieter** [comment]
Ping @adiko01

**Comment by enen92** [comment]
Please reopen if still interested in continuing the submission

**Pull request closed without merging**